lib/content-services/src/lib/common/services/nodes-api.service.ts

```diff
@@ -246,6 +257,27 @@ export class NodesApiService {
         return this.createNodeInsideRoot(name || this.randomNodeName(), nodeType, properties, path);
     }
 
+    /**
+     * Initiate a new request to calculate folder size.
+     *
+     * @param nodeId Node Id
+     * @returns The job id which can be used to track request status
+     */
+    initiateFolderSizeCalculation(nodeId: string): Observable<JobIdBodyEntry> {
+        return from(this.nodesApi.initiateFolderSizeCalculation(nodeId));
+    }
+
+    /**
+     * Gets the size of a folder.
+     *
+     * @param nodeId Node Id
+     * @param jobId Job Id
+     * @returns Folder details
+     */
+    getFolderSizeInfo(nodeId: string, jobId: string): Observable<SizeDetailsEntry> {
+        return from(this.nodesApi.getFolderSizeInfo(nodeId, jobId));
```

Comment:
What about handling some cases when the job is not done yet and user is trying to get the response? Are you planning to handle it here or rather in ACA when this will be called?